Use `\mathit` for multi-letter identifiers

EFanZh opened the pull request:
By default, in math environment, each letter is a separate identifier or variable, so a consecutive sequence of letters typically means multiplication of several variables. To represent a multi-letter identifier, it’s better to wrap it in `\mathit{}`. Without `\mathit{}`, the result identifier would have awkward spacings between each letter.

drupol commented:
Thanks again for your contribution!

Could you post an image before/after so we can visually see the changes you're trying to propose?

EFanZh commented:
Sure.

![image](https://github.com/hmemcpy/milewski-ctfp-pdf/assets/2160692/720800e7-fe86-4f06-97af-6024b41c9b05)

The left one is before this change, the right one is after the change. Note the spacing around the *l* letter in *coalg* and the *t* letter in *Store*.

Here is another one:

![image](https://github.com/hmemcpy/milewski-ctfp-pdf/assets/2160692/b553a39b-a6a6-4d3d-a972-80f71a6e511f)

The top one is before this change, the bottom one is after this change. Note spacings around *s* in *fst* and *n* in *snd*.

For other comparisons, here are the full PDFs before and after this change, you can compare each change yourself:

- [before.pdf](https://github.com/hmemcpy/milewski-ctfp-pdf/files/13064099/before.pdf)
- [after.pdf](https://github.com/hmemcpy/milewski-ctfp-pdf/files/13064101/after.pdf)

drupol commented:
I don't see the difference between the two images, except in the last line... Same thing for the linked PDFs :(

![image](https://github.com/hmemcpy/milewski-ctfp-pdf/assets/252042/18aabb0f-6269-4f9b-80e8-e92552d40bbe)

drupol commented:
I fail to see the changes for the e in get, the e in set and the t letter in Store.

Anyway, I got your point by looking at the the spacing around the l letter in coalg.

So, that's enough for me :)

drupol commented:
Thanks for your contribution :)

EFanZh commented:
Although it is a bit too late, here I have added some auxiliary lines for comparing the spacings:

![image](https://github.com/hmemcpy/milewski-ctfp-pdf/assets/2160692/d267cc4e-d102-4915-a47f-ba19cef0f6a7)

The *get* to *set* change is a typo that I accidentally introduced. I have fixed it in the latest commit.

This is the new PDF with the fix above: [after.pdf](https://github.com/hmemcpy/milewski-ctfp-pdf/files/13072562/after.pdf).
